.github/workflows/cicd.yml

```
@@ -34,16 +34,28 @@ jobs:
-        uses: appleboy/ssh-action@v0.1.10
+        uses: appleboy/ssh-action@2ead5e3
         with:
           host: ${{ secrets.EC2_HOST }}
           username: ${{ secrets.EC2_USERNAME }}
           key: ${{ secrets.EC2_SSH_KEY }}
+          capture_stdout: true
           script: |
-            PORT=$(sudo grep -oP 'server 127\.0\.0\.1:\K\d+' /etc/nginx/sites-available/default | head -n 1)
-            echo -n "$PORT"
+            sudo grep -oP 'server 127\.0\.0\.1:\K\d+' /etc/nginx/sites-available/default | head -n 1
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)